packages/nimble-components/src/toggle-button/tests/toggle-button-matrix.stories.ts

```diff
@@ -47,32 +52,45 @@ type CheckedState = (typeof checkedStates)[number];
 
 // prettier-ignore
 const component = (
+    [interactionName, interaction]: InteractionState,
     [iconVisible, labelVisible, endIconVisible]: PartVisibilityState,
     [checkedName, checked]: CheckedState,
     [disabledName, disabled]: DisabledState,
     [appearanceName, appearance]: AppearanceState
 ): ViewTemplate => html`
     <${toggleButtonTag}
+        class="${() => interaction}"
         appearance="${() => appearance}"
         ?disabled=${() => disabled}
         ?content-hidden=${() => !labelVisible}
         ?checked=${() => checked}
         style="margin-right: 8px; margin-bottom: 8px;">
             ${when(() => iconVisible, html`<${iconKeyTag} slot="start"></${iconKeyTag}>`)}
-            ${() => `${checkedName} ${appearanceName} Toggle Button ${disabledName}`}
+            ${() => `${interactionName} ${checkedName} ${appearanceName} Toggle Button ${disabledName}`}
             ${when(() => endIconVisible, html`<${iconArrowExpanderDownTag} slot="end"></${iconArrowExpanderDownTag}>`)}
     </${toggleButtonTag}>
 `;
 
 export const toggleButtonThemeMatrix: StoryFn = createMatrixThemeStory(
     createMatrix(component, [
+        interactionStates.slice(0, 1),
         partVisibilityStates,
         checkedStates,
         disabledStates,
         appearanceStates
     ])
 );
 
+export const toggleButtonInteractionsThemeMatrix: StoryFn = createMatrixThemeStory(
+    createMatrix(component, [
+        interactionStates.filter(x => x[0] && !x[0].includes('Active')),
```

Comment:
It isn't clear to me why we need to skip the `Active` state in these tests. Perhaps a comment could help.

Reply:
I think I made the mistake of thinking toggle button didn't have active styling, but it does, so I will include active states, too.